lib/core/src/Cardano/Pool/Metrics.hs

```diff
@@ -238,8 +266,25 @@ newStakePoolLayer db@DBLayer{..} nl tr = StakePoolLayer
             let tip = nodeTip ^. #slotId
             perfs <- liftIO $ readPoolsPerformances db epochLength tip
             combineWith (pure . sortByPerformance) distr prod perfs
-    }
-  where
+
+    findMetadata :: [PoolId] -> IO [Maybe StakePoolMetadata]
+    findMetadata poolIds = do
+        owners <- atomically $ mapM readStakePoolOwners poolIds
+        -- note: this will become simpler once we cache metadata in the database
+        let owners' = nub $ concat owners
+        url <- getRegistryZipUrl
+        let tr' = contramap (fmap MsgRegistry) tr
+        getStakePoolMetadata tr' url owners' >>= \case
+            Left _ -> do
+                logTrace tr MsgMetadataUnavailable
+                pure $ replicate (length poolIds) Nothing
```

Comment:
**[remark]**

Isn't `trace` maybe too low for this kind of error? We are not really expecting this to be unavailable, are we? 
I'd rather go for `Warning` or `Notice` for this one. Though `Warning` is maybe too strong because there is really nothing we can do about it.

Reply:
The log level is set in `instance DefineSeverity StakePoolLayerMsg`. I had made it Warning but actually now that you mention it, Notice is more suitable.

Reply:
Ho! Yes. I missed that sorry, got confused with `logTrace` :+1: